setup.py

```diff
@@ -291,10 +310,17 @@ def run(self):
     elif sys.platform == 'darwin':
         ext_modules.append(Extension('borg.platform_darwin', [platform_darwin_source]))
 
+
+def parse(root, describe_command=None):
+    return subprocess.check_output("git describe --tags --long").decode().strip()
```

Comment:
It still have to return something - maybe it should also write it to _version.py